docs/pages/beta/crm-beta/crm-beta.yaml

```diff
@@ -173,6 +173,19 @@ paths:
                   $ref: '#/components/schemas/IncludeFields'
                 cursor:
                   type: string
+                filters:
```

Comment:
don't forget to update the example

Reply:
since this is in the `list` endpoint, I was thinking it might make sense to leave it out of the example, so it's not construed as being required